docs/reference-1.4.0.html

```diff
@@ -9568,7 +9568,7 @@ <h3 id='polygon-method'>Methods</h3>
 	</tr>
 	</thead><tbody>
 	<tr id='polygon-togeojson'>
-		<td><code><b>toGeoJSON</b>(<nobr>&lt;Number&gt;</nobr> <i>precision</i>)</nobr></code></td>
+		<td><code><b>toGeoJSON</b>()</nobr></code></td>
```

Comment:
No need to remove this from the current docs. Reverting the patch shouldn't affect valid documentation from previous versions.